app/javascript/react/constants/restaurantsdata.json

```diff
@@ -0,0 +1,267 @@
+[
```

Comment:
I thought there was a json file that got deleted. Is this the same one or a new one that you want to keep?